common/changes/office-ui-fabric-react/schedule_time_secondary_2018-07-02-17-29.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Schedule time neutralTertiary -> neutralSecondary ",
```

Comment:
I'm not sure this comment here makes sense... Could you please reword this to better describe the change? E.g. "ContextualMenu: [...]"